### ISSUE TITLE
need better docs for csp

### ISSUE THREAD

**Issue by ralyodio** [primatejs/primate#41]
```
 http.csp

Default

{
// all content must come from own origin, excluding subdomains
"default-src": "'self'",
// styles must come from own origin, excluding subdomains
"style-src": "'self'",
// disallow <object>, <embed> and <applet> elements
"object-src": "'none'",
// disallow embedding
"frame-ancestors": "'none'",
// all form submissions must be to own origin
"form-action": "'self'",
// allow only own origin in <base>
"base-uri": "'self'",
}
```

this isnj't show us inline or eval

**Comment by ralyodio** [comment]
....or how to just whitelist everything so we can work.

**Comment by ralyodio** [comment]
this is how i whitelisted everything (order matters here):

		csp: {
			'script-src': "'unsafe-inline' 'self' 'unsafe-eval' *",
			'style-src': "'unsafe-inline' 'self' *",
			'img-src': "'unsafe-inline' 'self' *"
		}

**Comment by ralyodio** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy

**Comment by terrablue** [comment]
Thanks, we'll probably switch to more permissive defaults. We do use style-src and script-src ourselves, at least for style, we could switch to nonces.

**Comment by terrablue** [comment]
solved in [944779c](https://github.com/primatejs/primate/commit/944779c61cee5b67e66d674b0fdc7591b2c54346), default primate now has no csp
